apps/backend/src/modules/installation/installation.controller.ts

```diff
@@ -1,30 +1,22 @@
-import { Body, Controller, Delete, Get, Param, Post } from "@nestjs/common";
-import { CreateInstallationDto } from "./dto/create-installation.dto";
-import { Installation } from "./installation.entity";
-import { InstallationService } from "./installation.service";
+import { Controller, Delete, Get, Param } from '@nestjs/common';
+import { Installation } from './installation.entity';
+import { InstallationService } from './installation.service';
 
-@Controller("installations")
+@Controller('installations')
 export class InstallationController {
   constructor(private readonly installationService: InstallationService) {}
 
-  @Post()
-  async create(
-    @Body() createInstallationDto: CreateInstallationDto,
-  ): Promise<Installation> {
-    return await this.installationService.create(createInstallationDto);
-  }
```

Comment:
Removed create installation endpoint. All installations will now be handled using `/slack/install`